pkg/core/native/invocation_test.go

```diff
@@ -66,7 +66,9 @@ func TestNativeContract_Invoke(t *testing.T) {
 }
 
 func TestNativeContract_InvokeInternal(t *testing.T) {
```

Comment:
This test contains the code that was commented out. We need an issue to deal (enable/remove) this code.

Reply:
https://github.com/nspcc-dev/neo-go/issues/3743